<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Veeral-Patel" failures="4" tests="4" name="Test" time="50.022" errors="0" timestamp="18 Sep. 2022 19:19:45 GMT">
  <testcase classname="test1.LoginTest1" name="LoginWithSpecialCharTest" time="8.489">
    <failure type="java.lang.AssertionError" message="expected [Please enter a username or card number without special characters.] but found [This should be working, but something&amp;apos;s gone wrong. Please try again soon. [500.GENERIC]]">
      <![CDATA[java.lang.AssertionError: expected [Please enter a username or card number without special characters.] but found [This should be working, but something's gone wrong. Please try again soon. [500.GENERIC]]
at test1.LoginTest1.LoginWithSpecialCharTest(LoginTest1.java:42)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- LoginWithSpecialCharTest -->
  <testcase classname="test1.LoginTest1" name="LoginWithEmptyEmail" time="8.396">
    <failure type="java.lang.AssertionError" message="expected [Please enter your username or card number.] but found [Please enter your Username or Access Card number.]">
      <![CDATA[java.lang.AssertionError: expected [Please enter your username or card number.] but found [Please enter your Username or Access Card number.]
at test1.LoginTest1.LoginWithEmptyEmail(LoginTest1.java:53)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- LoginWithEmptyEmail -->
  <testcase classname="test1.LoginTest1" name="LoginWithEmptyPassword" time="8.441">
    <failure type="java.lang.AssertionError" message="expected [Please enter your password.] but found [Please enter your password]">
      <![CDATA[java.lang.AssertionError: expected [Please enter your password.] but found [Please enter your password]
at test1.LoginTest1.LoginWithEmptyPassword(LoginTest1.java:63)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- LoginWithEmptyPassword -->
  <testcase classname="test1.LoginTest1" name="LoginWithwrongemailpasswordTest" time="8.507">
    <failure type="java.lang.AssertionError" message="expected [Please check your card number / username or password and try again.] but found [This should be working, but something&amp;apos;s gone wrong. Please try again soon. [500.GENERIC]]">
      <![CDATA[java.lang.AssertionError: expected [Please check your card number / username or password and try again.] but found [This should be working, but something's gone wrong. Please try again soon. [500.GENERIC]]
at test1.LoginTest1.LoginWithwrongemailpasswordTest(LoginTest1.java:74)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- LoginWithwrongemailpasswordTest -->
</testsuite> <!-- Test -->
